pkg/api/alertmanager.go

```diff
@@ -241,8 +241,17 @@ func (c *GettableApiAlertingConfig) Type() (backend Backend) {
 	return
 }
 
+// Config is the top-level configuration for Alertmanager's config files.
```

Comment:
Why do we need to copy this struct over? Can't we continue to reference it as `config.Config`?

Reply:
The difference is the `yaml:"-"` tag for the `Receivers` field because otherwise it conflicts with the `Receivers` field of the `PostableApiAlertingConfig` struct.